targets/TARGET_NUVOTON/TARGET_M2351/TARGET_M23_NS/device/TARGET_TFM/cmsis_nvic_virtual.c

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2018-2020, Nuvoton Technology Corporation
```

Comment:
2019 instead of 2020 (some files have this one as well).

Reply:
Not familiar with copyright claim. Which one is better, 2018-2020, 2019-2020, or 2019?

Reply:
just use the year this was created/edit. we are not yet in 2020

Reply:
2018-2019 would be fine here

Reply:
@0xc0170 Fixed.